app/config/ApplicationConfig.scala

```diff
@@ -94,6 +94,11 @@ class ApplicationConfig @Inject()(config: Configuration, runMode: RunMode) exten
   val ppnsSandboxApiKey = getConfString("push-pull-notifications-api-sandbox.api-key", "")
   val ppnsSandboxAuthorizationKey = getConfString("push-pull-notifications-api-sandbox.authorizationKey", "")
 
+  // Fraud Prevention Headers
+  lazy val fraudPreventionLinkVisible = getConfigDefaulted("fraudPreventionLinkVisible", false)
```

Comment:
Can we loose the defaulted.  We're trying to move to HOCON so these go in application.conf instead and blow up if you loose them.  I know the other config stull has defaults.  But they are also yuk.